tests/JoinSqlTest.php

```diff
@@ -431,21 +431,21 @@ public function testDoubleJoin()
         $j_country = $j_contact->join('country');
         $j_country->addField('country_name', ['actual' => 'name']);
 
-        $m_u->load(10);
-        $m_u->delete();
+        $m_u2 = (clone $m_u)->load(10);
```

Comment:
last 3 tests are failing - @georgehristov, you are the data hero, do you know where the issue is and how to fix the tests?